streampark-common/src/main/java/org/apache/streampark/common/util/ExceptionGetter.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.streampark.common.util;
+
+import javax.annotation.Nonnull;
+
+/** A interface to define the action to create a exception with specified error message. */
+public interface ExceptionGetter {
+
+  @Nonnull
+  Exception get(String errorMsg);
+}
```

Comment:
could we move the `org.apache.streampark.console.base.exception` package into `streampark-common` module ? 
It would be better merge the condition logic and throw exceptions logic , which is useful to unify the behavior of `Utils`. But there's dependency conflict in the packages and modules when I want use throw `ApiAlertException` instead of passing by `ExceptionGetter`.

Reply:
thanks for your contribution, I don't quite agree with this method, we just need to add method in ApiAlertException and ApiDetailException, such as:

```
  public static void throwIfNull(Object object, String message) {
         if (object == null) {
             throw new ApiAlertException(message);
         }
     }
```
how to use:
```
ApiAlertException.throwIfNull(executionMode, "executionMode cannot be null");
```

Would this be simpler?

Reply:
Yes. There's nothing better. 
It's my original strategy, too. 
Thanks for your quick-reply.

Reply:
> Yes. There's nothing better. It's my original strategy, too. Thanks for your quick-reply.

👌👍